acs5countypoverty.ipynb

```diff
@@ -0,0 +1,145 @@
+{
+ "cells": [
```

Comment:
If we create a `.py` file, I'm thinking it is redundant to also commit the jupyter notebook and creates the possibility that they will be out of sync. Thoughts?

Reply:
I will be using it to get the zip code data. It may be helpful to keep it if anyone else wants to explore the data.

Reply:
I think if someone else wants to explore the data, they could just copy the script into a jupyter notebook? Just to give a bit more context about where I'm coming from: I want to maintain a single "source of truth". Currently, we have 3 sources of truth: 
1. the .py script
2. the jupyter notebook
3. the final json

But we can create `#2` and `#3` from `#1`, so we should only commit `#1` to the repo. Does that make sense?

Reply:
What if we put the jupyter notebook in a shared CoLab account? https://colab.research.google.com/notebooks/intro.ipynb#recent=true

Reply:
Not sure how the colab account works with protecting api keys. Copy and pasting from the python script is fine.

I'll add .json and .ipynb to the gitignore.

Reply:
Good point, I’d have to look into how to safely store api keys with CoLab 🤔  I guess we can see how annoying it is to copy and paste the script back into a jupyter notebook when you want to investigate... if it gets too annoying we can figure something else out